### ISSUE TITLE
Rates returning nil on both Production & Test API Keys

### ISSUE THREAD

**Issue by Archetapp** [welbesw/easypost-swift#3]
Hi There!

Been using this awesome framework for the last couple of weeks in testing. When I first set it up, everything worked great, rates were returning & I was able to but shipments no problem. Now, a few weeks later, the rates are always returning nil. All the rest seems to be uploading properly... Any idea why the change? Here's my code:
```
func createShipment(addressFrom : addressStruct, addressTo : addressStruct, productToShip : packageInfoStruct, completion : @escaping (_ result : NSError?) -> ()) {
        let apiToken = "XXXXXXXXXXXXXX"
        EasyPostApi.sharedInstance.setCredentials(apiToken, baseUrl: "https://api.easypost.com/v2/")
        EasyPostApi.sharedInstance.postShipment(EasyPostAddress.init(jsonDictionary: addressTo.convertToAnyObject()!), fromAddress: EasyPostAddress.init(jsonDictionary: addressFrom.convertToAnyObject()!), parcel: EasyPostParcel.init(jsonDictionary: productToShip.convertToAnyObject()!)) { (shipment) in
            DispatchQueue.main.async {
                switch(shipment) {
                case .success(let shipment):
                    
                    print("Successfully posted shipment.")
                    
                    if let id = shipment.id {
                        print(shipment.rates)
                        EasyPostApi.sharedInstance.buyShipment(id, rateId: (shipment.rates.first?.id)!, completion: { (response) in
                            print(response)
                        })
                    }
                    
                case .failure(let error):
                    print("Error posting shipment: \((error as NSError).localizedDescription)")
                }
            }
        }
    }
```
I put a print statement in your code to figure out what the JSON it's returning looks like : 

```
...
"rates": <__NSArray0 0x1c400fe40>(

)
...
```

Any help would be greatly appreciated! This doesn't seem to be an issue with your code, more of an issue with something with EasyPost or myself, but I'd love some help if you can. Thanks! :]

**Comment by welbesw** [comment]
Hello. Are you using the latest version tag 1.1.1 of the code?  I believe there was an issue with array types returned on one interim version.

**Comment by Archetapp** [comment]
Yes I am, I updated pods this morning. 👍

**Comment by Archetapp** [comment]
![screen shot 2018-01-02 at 10 52 06 pm](https://user-images.githubusercontent.com/8635253/34510397-d2578200-f010-11e7-90a5-d5480662c78a.png)

So, I did just notice this error in EasyPost, I've now added billing & it doesn't seem to work still. I'll give it another go tomorrow to see if it just takes time to fix or not. I'm pretty sure the billing is what's causing it though. I'll get back to ya. :]
